<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diag_lipsync</class>
 <widget class="QDialog" name="diag_lipsync">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>355</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lip Sync</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_engine">
     <property name="text">
      <string>Engine:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_engine">
     <item>
      <property name="text">
       <string>1.0</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_format">
     <property name="text">
      <string>Format:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_format">
     <item>
      <property name="text">
       <string>PDA 2.00/Future Tone f mots</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_language">
     <property name="text">
      <string>Language:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox_language">
     <item>
      <property name="text">
       <string>English (United States)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>English (United Kingdom, RP)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Italian (Italy)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spanish (Spain)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spanish (Latin America)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Portuguese (Portugal)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Portuguese (Brasil)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>French (France)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>French (Belgium)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>German (Germany)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Latin script [aeioubpmn]</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_espeak_voice">
     <property name="text">
      <string>espeak voice:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_espeak_voice">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>en-us</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_performer">
     <property name="text">
      <string>Performer:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="2">
    <widget class="QSpinBox" name="spinBox_performer">
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkBox_delete_existing">
     <property name="text">
      <string>Delete existing</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_offset">
     <property name="title">
      <string>Offset (msec)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_offset_before">
        <property name="text">
         <string>Before:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_offset_after">
        <property name="text">
         <string>After:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="spinBox_offset_before">
        <property name="minimum">
         <number>-10000</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_offset_after">
        <property name="minimum">
         <number>-10000</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_close_mouth">
     <property name="title">
      <string>Close mouth</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_close_mouth_p">
        <property name="text">
         <string>[p]</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_close_mouth_n">
        <property name="text">
         <string>[n]</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_close_mouth_end">
        <property name="text">
         <string>End</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_close_mouth_b">
        <property name="text">
         <string>[b]</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QCheckBox" name="checkBox_close_mouth_m">
        <property name="text">
         <string>[m]</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>diag_lipsync</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>diag_lipsync</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
